lumigator/backend/backend/tests/conftest.py

```diff
@@ -463,8 +463,7 @@ def simple_infer_template():
             "revision": "{revision}",
             "use_fast": "{use_fast}",
             "trust_remote_code": "{trust_remote_code}",
-            "torch_dtype": "{torch_dtype}",
-            "max_new_tokens": 500
+            "torch_dtype": "{torch_dtype}"
```

Comment:
Does removing it from this config break anything? Why was `max_new_tokens` here?

Reply:
It appears to have been an accidental artifact left from https://github.com/mozilla-ai/lumigator/pull/762/files#diff-26ce53c706eccca82798fff14bfc531b57789138468d24dee14552943c640574?

max_new_tokens was added but then didn't appear to be used anywhere in the final version of the code that was merged.